app/matplotlib.ipynb

```diff
@@ -11,10 +11,8 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
-   "outputs": [],
    "source": [
-    "# import matplotlib.pyplot as plt\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
The `%matplotlib inline` magic command is specific to Jupyter notebooks and ensures that plots are displayed inline. Ensure that this is necessary for your environment, as it might not be needed in all cases.